components/tools/OmeroPy/test/unit/clitest/bulk_import/test_cols/bulk.tsv

```diff
@@ -0,0 +1,2 @@
+meta_one	1.fake
+meta_two	2.fake
```

Comment:
This `tsv` file references two fake files but only one exists in the directory. This makes no difference to the unit test but is inconsistent as an example.